.github/workflows/test.yml

```diff
@@ -626,3 +626,29 @@ jobs:
       - name: Test ${{ matrix.pkg }} Build
         run: |
           nix develop -c sh -c "cd pkg/${{ matrix.pkg }} ; zig build test"
+
+  test-debian-12:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: ["linux/amd64", "linux/arm64"]
```

Comment:
We probably only need to test amd64, I can't see a scenario where one works and one fails since what we're really protecting against with this test is package versions.

Reply:
Fixed.